pkg/p2p/wire/topics/topics.go

```diff
@@ -40,6 +40,11 @@ const (
 	Agreement      Topic = "blockagreement"
 	StartConsensus Topic = "startconsensus"
 
+	// Transaction topics
```

Comment:
Are these topics being used anywhere?

Reply:
Same goes for this, leftover code. Will remove it